src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/StraxCoinviewRule.cs

```diff
@@ -10,17 +13,33 @@ namespace Stratis.Bitcoin.Features.Consensus.Rules.CommonRules
     /// </summary>
     public sealed class StraxCoinviewRule : PosCoinviewRule
     {
+        private ISmartContractCoinViewRuleLogic logic;
+
         // 50% of the block reward should be assigned to the reward script.
         // This has to be within the coinview rule because we need access to the coinstake input value to determine the size of the block reward.
         public static readonly int CirrusRewardPercentage = 50;
 
+        /// <summary>
+        /// This constructor is used by legacy tests that do not exercise SC logic.
+        /// </summary>
+        public StraxCoinviewRule()
```

Comment:
maybe better to remove this constructor and instead provide null in tests that were using it

Reply:
done